addons/binding/org.openhab.binding.nibeuplink/utils/README.md

```diff
@@ -0,0 +1,20 @@
+# Usefull commands to create files used by the binding from Nibe ModbusManager CSV files
+
+Command to convert Nibe CSV file character set to UTF-8:
+ ```iconv -f iso-8859-1 -t utf-8 F1X45.csv > F1X45_utf8.csv```
```

Comment:
I don't know what the policy is to have these script in here. But the input file `F1X45.csv` is not part of it the scripts are only usable by someone having the input file and make having these script here of less use in general. Although I can understand someone updating the binding can be use these scripts (btw having a javax annotation compile time generator would remove the need for these scripts. But that's probably something too much..)

Reply:
I could remove the whole "utils" folder as it is not needed by the binding itself. But I would like to keep these util scripts as close as possible to the binding as it is quite useful for updating the channels from the CSVs provided by NIBE.